env_support/cmake/custom.cmake

```
@@ -41,10 +42,15 @@ if ( LV_USE_KCONFIG )
   # and autoconf.h, which will be used by lv_conf_kconfig.h
   include(${CMAKE_CURRENT_LIST_DIR}/kconfig.cmake)
 
-  set(LV_KCONFIG_IGNORE OFF)
+  # If KConfig is used - creating CMAKE variables from lv_conf_internal.h is required,
+  # so PCPP is used to pre-process lv_conf_internal.h
+  set(LV_CMAKE_CREATE_VARIABLES TRUE)
```

```suggestion
  set(LV_CMAKE_CREATE_CONF_VARS ON)
```

Should be this right ?

---

True or on is okay, as long as we keep things consistent

The variable name is wrong tho isn't it?